packages/autorest.python/samples/specification/multiapi/generated/azure/__init__.py

```diff
@@ -0,0 +1 @@
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore
```

Comment:
Old logic misses the general __init__ file although it doesn't influence runtime behavior.